modules/utxo-lib/src/noble_crypto.ts

```diff
@@ -0,0 +1,76 @@
+import * as createHash from 'create-hash';
+import * as necc from '@noble/secp256k1';
+import { MuSigFactory, MuSig } from '@brandonblack/musig';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore base_crypto is exported as a subPath export, ignoring since compiler complains about importing like this
+import * as base_crypto from '@brandonblack/musig/base_crypto';
+
+function toBigInt(b: Uint8Array | Buffer): bigint {
+  return Buffer.from(b).readBigUint64BE();
+}
+
+const crypto = {
+  ...base_crypto,
+  pointMultiplyUnsafe: (p: Uint8Array, a: Uint8Array, compress: boolean): Uint8Array | null => {
+    try {
+      const product = necc.Point.fromHex(p).multiplyAndAddUnsafe(necc.Point.ZERO, toBigInt(a), BigInt(1));
+      if (!product) return null;
+      return product.toRawBytes(compress);
+    } catch {
+      return null;
+    }
```

Comment:
why are we doing these broad catch-alls instead of propagating the error?

the interface states 

https://github.com/brandonblack/musig-js/blob/master/index.ts#L248-L257

> null if the result is the point at infinity.

But your implementation may return `null` for any number of reasons

Reply:
I am using [this](https://github.com/brandonblack/musig-js/blob/b2daacfa72e2488dab0f40e470a8ebd6ae2937bf/test/utils.ts#L86-L96) impl of the crypto interface.

I'll look into returning null only when it's point is at infinity.

Reply:
This is, roughly, to drag tiny- and noble- secp256k1s together. Because tiny is a native / wasm wrapper, it doesn't through, but returns null on failure. It's still the more common implementation, so I dragged noble in that direction instead of the reverse.